pkg/controller/iam/auditconfig/iamauditconfig_controller.go

```diff
@@ -156,6 +157,12 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		}
 		return reconcile.Result{}, err
 	}
+	// r.Get() overrides the TypeMeta to empty value, so need to configure it
```

Comment:
Did `Get()` actually clear it, or just not set it?

Reply:
It cleared it. I configured TypeMeta before the Get call and it got removed after the Get call.

Reply:
TIL!